.github/workflows/coder.yaml

```diff
@@ -274,6 +274,9 @@ jobs:
           export PATH=${PATH}:$(go env GOPATH)/bin
           make --output-sync -j -B fmt
 
+      - name: Check for unstaged files
+        run: ./scripts/check_unstaged.sh
```

Comment:
I thinks this is good, should we also consider doing a CI conditional for `terraform fmt -check` like we do for prettier and shfmt?

Reply:
Sorry, I noticed your comment after merging. I can iterate on this if you like, but I assume that `check_unstaged.sh` as an ultimate check should work here well.

Reply:
No worries, I'll leave it up to you. Ultimately this check is great to have as a catch-all, the other is just a minor improvement which doesn't bring much benefit.